src/xmipp/bindings/python/envs_DLTK/xmipp_graph.yml

```diff
@@ -0,0 +1,13 @@
+name: xmipp_graph
```

Comment:
Why this name?
Add as comment the protocol is ussing this enviroment (check the others environment *.yml files)

Reply:
The name relates to graph algorithms. I will add a list of programs that use it (xmipp_graph_max_cut)